packages/babel-plugin-named-asset-import/.npmignore

```diff
@@ -0,0 +1,2 @@
+*.spec.js
```

Comment:
Please use the `include` property in `package.json` instead of an ignore file.

Reply:
deleted